styleguide/template.md

```diff
@@ -129,20 +131,30 @@ When linking to an API, you can use its unique identifier (UID) that is auto-gen
 
 You can use one of the following syntax:
 
-1. Markdown link: `[link_text](xref:UID)`
-2. Auto link: `<xref:UID>`
-3. Shorthand form: `@UID`
+1. Auto-link: `<xref:UID>` or `<xref:UID?displayProperty=fullName>`
+
+ The displayProperty query parameter produces a fully qualified link text. Default behavior is to produce a link text that shows only the member or type name.
+ 
+2. Markdown link: `[link_text](xref:UID)`
+
+ Use only when the auto-link behavior is not the desired display. 
 
-- Example: `@System.String`
-- Example: `[String class](xref:System.String)` 
+Examples:
+- `<xref:System.String>` renders as [String](https://docs.microsoft.com/dotnet/api/system.string)
+- `<xref:System.String?displayProperty=fullName>` renders as [System.String](https://docs.microsoft.com/dotnet/api/system.string)
+- `[String class](xref:System.String)` renders as [String class](https://docs.microsoft.com/dotnet/api/system.string)
 
 For more information about using this notation, see [Using cross reference](https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html#using-cross-reference).
 
-> Right now, there is no easy way to find the UIDs. The best way to find the UID for an API is to search for it in [the `xml` directory of this repo](../xml), then looking at the DocId signature and ignoring its first two characters (letter and colon). We're working on having a better system in the future.
+> Right now, there is no easy way to find the UIDs. The best way to find the UID for an API is to view the source for the API page you want to link to and find the ms.assetid value. Individual overloads values are not shown on the source. We're working on having a better system in the future.
+
+When the UID contains the special characters \`, \# or \*, the UID value needs to be HTML encoded as `%60`, `%23` and `%2A` respectively. You'll sometimes see parentheses encoded but it's not a requirement.
```

Comment:
Possibly also explain `*` or `%2A` are used for overloaded methods.

Reply:
I should've added. My bad.